templates/_helpers.tpl

```diff
@@ -135,7 +135,7 @@ go:
   version: v1.148.6
   # Used for grpcx
 - name: github.com/getoutreach/services
-  version: v1.175.1
+  version: v1.218.0
```

Comment:
Updated this version to make sure to get a minimum services version without `services/pkg/k8s/*`.